debian/changelog

```diff
@@ -1,7 +1,9 @@
-
 opx-platform-config (1.5.0) unstable; urgency=medium
```

Comment:
(1) Is it really appropriate to increment the second digit of the package version? Check with the repo owners (Howard/Atanu/Garrick)

(2) You might run into a conflict with @hpersh 's upcoming PR:
https://github.com/open-switch/opx-platform-config/pull/47

Reply:
If we're keeping up with OS10 version # then we should use the '+opx#' convention.